pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -141,7 +141,9 @@ public void getLeaderBroker(@Suspended final AsyncResponse asyncResponse) {
         validateSuperUserAccessAsync().thenAccept(__ -> {
                     LeaderBroker leaderBroker = pulsar().getLeaderElectionService().getCurrentLeader()
                             .orElseThrow(() -> new RestException(Status.NOT_FOUND, "Couldn't find leader broker"));
-                    BrokerInfo brokerInfo = BrokerInfo.builder().serviceUrl(leaderBroker.getServiceUrl()).build();
+                    BrokerInfo brokerInfo = BrokerInfo.builder()
+                            .serviceUrl(leaderBroker.getServiceUrl())
+                            .lookupServiceAddress(leaderBroker.getLookupServiceAddress()).build();
```

Comment:
Since the lookup service address is the address used in the load manager to identify the broker, maybe we can name it `brokerId`?

Reply:
> Can we avoid this change to the public API? The newly added field lookupServiceAddress might confuse users. Users don't know the value localhost:8080 is HTTP lookup service or Binary lookup service.\n\nBTW, the serviceUrl is also not a good name. I think it should be webServiceUrl

It is necessary to add a field here and leave the serviceUrl alone, since that would break backwards compatibility or mixed runtime version compatibility.

I'll rename "brokerLookupAddress" to "brokerId" everywhere in the code base as suggested so that we reduce confusion.